app/src/test/kotlin/io/eugenethedev/taigamobile/repositories/TasksRepositoryTest.kt

```diff
@@ -0,0 +1,945 @@
+                    actual = story.title
+                )
+                assertEquals(
+                    expected = testEpicsUserStories[index].assignedTo?.fullName,
+                    actual = story.assignee?.fullName
+                )
+                assertEquals(
+                    expected = testEpicsUserStories[index].isClosed,
+                    actual = story.isClosed
+                )
+            }
+        }
+    }
+
+    @Test
+    fun `test get user story tasks`() = runBlocking {
+        tasksRepository.getAllUserStories().forEach { story ->
+            val tasks = tasksRepository.getUserStoryTasks(story.id).sortedBy { it.title }
+            val testTask = TestData.projects[0].userstories
+                .find { it.title == story.title }
+                ?.tasks
+                ?.sortedBy { it.title }
+
+            assertEquals(
+                expected = testTask?.size,
+                actual = tasks.size
+            )
+            if (testTask != null) {
+                tasks.forEachIndexed { index, task ->
+                    assertEquals(
+                        expected = testTask[index].title,
+                        actual = task.title
+                    )
+                    assertEquals(
+                        expected = testTask[index].assignedTo?.fullName,
+                        actual = task.assignee?.fullName
+                    )
+                    assertEquals(
+                        expected = testTask[index].isClosed,
+                        actual = task.isClosed
+                    )
+                }
+            }
+        }
+    }
+
+    @Test
+    fun `test get issues`() = runBlocking {
+        val issues = tasksRepository.getIssues(1, FiltersData()).sortedBy { it.title }
+        val testIssues = TestData.projects[0].issues.sortedBy { it.title }
+
+        assertEquals(
+            expected = testIssues.size,
+            actual = issues.size
+        )
+        issues.forEachIndexed { index, issue ->
+            assertEquals(
+                expected = testIssues[index].title,
+                actual = issue.title
+            )
+            assertEquals(
+                expected = testIssues[index].assignedTo?.fullName,
+                actual = issue.assignee?.fullName
+            )
+            assertEquals(
+                expected = testIssues[index].isClosed,
+                actual = issue.isClosed
+            )
+        }
+    }
+
+    @Test
+    fun `test get common task`() = runBlocking {
+        val userStories = TestData.projects[0].userstories + TestData.projects[1].userstories
+        val tasks =
+            TestData.projects[0].sprints.flatMap { it.tasks } + TestData.projects[1].sprints.flatMap { it.tasks }
+        val epics = TestData.projects[0].epics + TestData.projects[1].epics
+        val issue = TestData.projects[0].issues + TestData.projects[1].issues
+        val dataForTest = hashMapOf(
```

Comment:
Regular `mapOf()` would do the trick, since we do not need `HashMap` especially.